src/components/DataTable/__tests__/DataTable-test.js

```diff
@@ -1,41 +1,41 @@
 jest.dontMock('../DataTable');
-jest.dontMock('../util');
+jest.dontMock('../../../core/util');
 
 import React from 'react';
 import TestUtils from 'react-addons-test-utils';
 
 const DataTable = require('../DataTable');
 
-const testData = {
-  "tasmin_Amon_CanESM2_historical_r1i1p1_19610101-19901231": 
-  {
+const testData = [ 
+  { 
     "median": 278.34326171875,
```

Comment:
You're just testing for trutiness down below, so why bother with listing a bunch of data points to the 10th decimal place? It just ends up being noise. If your "test data" isn't actually required to run the test, then perhaps you should consider whether it even needs to be there? Corollary: if you test data doesn't need to be there, then what are you actually testing?